components/org.wso2.carbon.identity.inbound.ldap/src/main/java/org/wso2/carbon/identity/inbound/ldap/impl/ConfigurationConstants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.identity.inbound.ldap.impl;
+
+/**
+ * Class which encapsulates, most of the configuration parameters used in apacheds implementation.
+ */
+public class ConfigurationConstants {
+
+    public static final String ADMIN_PASSWORD_ALGORITHM = "SHA";
```

Comment:
unused constant. Algorithm need to mention the bit length - say SHA256.

Reply:
I used this constant for ApacheLDAPServer.java  file (changeConnectionUserPassword method at line num 231).